build/openwisp_websocket/daphne.conf

```diff
@@ -14,3 +14,5 @@ autorestart=true
 stopsignal=INT
 stdout_logfile=/dev/stdout
 stderr_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
```

Comment:
why are these needed? If there's a reason, we may need to add a comment because it's not clear to me

Reply:
It's required because otherwise we get `error: invalid seek`, it's not a fatal error that stops the service but it ruins the logs.
I will add a comment here. :smile:

Reply:
ok, let me know when is done

Reply:
Added a commit above this line. :smile: